lib/card_items/countdown_timer_card.dart

```diff
@@ -0,0 +1,59 @@
+import 'dart:async';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+final hackathonDate = DateTime.utc(2022, 4, 8, 14);
```

Comment:
@rsquared226 I forgot what time you said I should set this too

Reply:
April 8th 2pm EST 👺😘🥸🥳🤓🥶😴🥱😵💩👽🤲💋🗣👁👄👁👣👩‍🦳🕵️‍♂️👨🏼‍💻🦹‍♀️🧖‍♀️💆🏾‍♂️👩‍❤️‍💋‍👨👨‍👨‍👧‍👧👩‍👧‍👧👩‍👦‍👦👩‍👦‍👦👙🌂